### PR TITLE
Dockerize CinemaService app

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_DB=POSTGRES_DB
+POSTGRES_USER=POSTGRES_USER
+POSTGRES_PASSWORD=POSTGRES_PASSWORD
```
Comment (NikOneZ1) [inline .env.sample:4]: Add empty line

File: dockerfile
Comment (NikOneZ1) [inline dockerfile:1]: Dockerfile name must begin with a capital letter

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 1 above)
Comment (goldenuni) [reply .env.sample]: @NikOneZ1 +

File: dockerfile
Comment (goldenuni) [reply dockerfile:1]: @NikOneZ1 +

**Review by NikOneZ1 — APPROVED** [review]
Well done!

**Pull request closed without merging**